src/Query/Builder.php

```diff
@@ -23,11 +23,10 @@
 use Illuminate\Contracts\Support\Arrayable;
 use Illuminate\Database\Query\Builder as BaseBuilder;
 use Illuminate\Support\Arr;
-use Illuminate\Support\Collection;
 use LogicException;
 
 /**
- * @method Collection<int, array<array-key, mixed>> get($columns = ['*'])
+ * @template TConnection of Connection
```

Comment:
This generics is not necessary.

Reply:
Without this Generics, type checking in the dependent in-house libraries will not work properly. This will be explained in the our company Slack.

Reply:
I found out that it probably wasn't a problem, so I removed it.